### DISCUSSION TITLE
AlpineJS is bundled with Livewire doc need to update

### DISCUSSION THREAD

**Discussion by nasirkhan** [livewire/livewire#6526] [1. Help]
AlpineJS is bundled with Livewire section that need to be fixed, the code is not visible. 

![image](https://github.com/livewire/livewire/assets/396987/f9179056-2492-48bb-8cd4-a59630dc8c21)

**Comment by joshhanley** [comment]
@nasirkhan thanks for reporting! PRs to the docs are welcome! 🙂 https://github.com/livewire/livewire/tree/main/docs

**Reply by nasirkhan** [reply]
how can I preview/test the doc locally as the doc site?

**Reply by joshhanley** [reply]
@nasirkhan you can't as the docs site is closed source, just make the changes in the markdown files 🙂

**Comment by nasirkhan** [comment]
submitted a PR, which might fix the issue. https://github.com/livewire/livewire/pull/6534

**Reply by joshhanley** [reply]
Thanks!
